src/ctia/logging.clj

```diff
@@ -0,0 +1,15 @@
+(ns ctia.logging
+  (:require [clojure.tools.logging :as log]
+            [clojure.core.async :as a :refer [go <! chan tap]]
+            [ctia.events :as e]
+            [schema.core :as s])
+  (:import [clojure.core.async.impl.protocols Channel]))
+
+(s/defn log-channel :- Channel
+  "Logging an event channel indefinitely. Returns the channel for a go loop that never ends."
+  [{m :mult :as ev} :- e/EventChannel]
+  (let [events (chan)]
+    (tap m events)
+    (go (loop [ev (<! events)]
```

Comment:
Please ensure that you shutdown gracefully.

http://stackoverflow.com/questions/20485188/gracefully-exit-a-clojure-core-async-go-loop-on-kill

Try running this and then killing your process.  if it hangs for a bit, remedy as above.  If it dies immediately, you are golden.